### PR TITLE
Set of fixes for the Spelling Tool.

### PR DESCRIPTION
**Pull request by samgnaniah**
These are fixes done using the Spelling Tool.

### CODE REVIEW

File: en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This section explains how to attach a custom workflow to the user signup operati
 #### Configuring the Enterprise Integrator
 
 !!! note
-    Follow this sub section, only if you will be using the **Entreprise Integrator(EI)** for the business process tasks. If not please refer the sub section for [Configuring the Business Process Server.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-business-process-server)
+    Follow this sub section, only if you will be using the **Enterprise Integrator(EI)** for the business process tasks. If not please refer the sub section for [Configuring the Business Process Server.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-business-process-server)
```
Comment (Mariangela) [inline en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md:27]: This is not part of this exercise, but this link will appear in a page not found as there is no {{base_path}} in front of the path.

File: en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ This section explains how to attach a custom workflow to the user signup operati
 #### Configuring the Business Process Server
 
 !!! note
-    Follow this sub section, only if you will be using the **Business Process Server(BPS)** for the business process tasks. If not please refer the sub section for [Configuring the Entreprise Integrator.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-enterprise-integrator)
+    Follow this sub section, only if you will be using the **Business Process Server(BPS)** for the business process tasks. If not please refer the sub section for [Configuring the Enterprise Integrator.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-enterprise-integrator)
```
Comment (Mariangela) [inline en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md:101]: This is not part of this exercise, but this link will appear in a page not found as there is no {{base_path}} in front of the path.

File: en/docs/reference/mediators/db-report-mediator.md [hunk 3]
```diff
@@ -368,7 +368,7 @@ This example demonstrates simple database write operations. The DB Report mediat
 
 ### Database write operation within a transaction
 
-In this example, `         <transaction action="new"/>        ` is a Transaction Mediator configuration that starts a new transaction. The DBReport mediator configuration performs a few write operations incuding deleting records when the name matches a specific value derived via an
+In this example, `         <transaction action="new"/>        ` is a Transaction Mediator configuration that starts a new transaction. The DBReport mediator configuration performs a few write operations including deleting records when the name matches a specific value derived via an
```
Comment (Mariangela) [inline en/docs/reference/mediators/db-report-mediator.md:371]:
```suggestion
In this example, `<transaction action="new"/>` is a Transaction Mediator configuration that starts a new transaction. The DBReport mediator configuration performs a few write operations including deleting records when the name matches a specific value derived via an
```

File: en/docs/reference/mediators/db-report-mediator.md [hunk 4]
```diff
@@ -368,7 +368,7 @@ This example demonstrates simple database write operations. The DB Report mediat
...
 expression as well as a few insertions. Once the database operations are
 complete, they are committed via
 `         <transaction action="commit"/>        ` , which is another
```
Comment (Mariangela) [inline en/docs/reference/mediators/db-report-mediator.md:374]:
```suggestion
`<transaction action="commit"/>` , which is another
```

File: en/docs/reference/mediators/fastxslt-mediator.md [hunk 5]
```diff
@@ -130,7 +130,7 @@ Pass the following XML payload using SOAP UI.
     example, the following payload passes the `         drinkName        `
     as 'Coffee'. The style sheet traverses through the incoming payload and
     finds the `         <lunch>        ` elements, which contains 'Coffee'
-    as `         drinkName        ` . When it finds matching ectries, it
+    as `         drinkName        ` . When it finds matching entries, it
```
Comment (Mariangela) [inline en/docs/reference/mediators/fastxslt-mediator.md:133]:
```suggestion
    as `drinkName` . When it finds matching entries, it
```

File: en/docs/reference/mediators/fastxslt-mediator.md [hunk 6]
```diff
@@ -130,7 +130,7 @@ Pass the following XML payload using SOAP UI.
...
     adds the prices of those elements under a new
     `         <Payment>        ` element. Therefore, when the message flow
```
Comment (Mariangela) [inline en/docs/reference/mediators/fastxslt-mediator.md:135]:
```suggestion
    `<Payment>` element. Therefore, when the message flow
```

File: en/docs/reference/mediators/property-mediator.md [hunk 7]
```diff
@@ -184,7 +184,7 @@ For example, the following Synapse configuration retrieves the
 ```
 
 !!! Info
-    You can use the following syntaxes to read properties or resources stored in the `         gov        ` or `         conf        ` Registries. When specifying the path to the resource, do not give the absolute path. Instead, use the `         gov        ` or `         conf        ` prefixes.
+    You can use the following syntax to read properties or resources stored in the `         gov        ` or `         conf        ` Registries. When specifying the path to the resource, do not give the absolute path. Instead, use the `         gov        ` or `         conf        ` prefixes.
```
Comment (Mariangela) [inline en/docs/reference/mediators/property-mediator.md:187]:
```suggestion
    You can use the following syntax to read properties or resources stored in the `gov` or `conf` Registries. When specifying the path to the resource, do not give the absolute path. Instead, use the `gov` or `conf` prefixes.
```

File: en/docs/reference/mediators/script-mediator.md [hunk 8]
```diff
@@ -3,7 +3,7 @@
 The **Script Mediator** is used to invoke the functions of a variety of scripting languages such as JavaScript, Groovy, or Ruby.
 
 !!! Note
-    The Micro Integrator uses Rhino engine to execute JavaScripts. Rhino engine converts the script to a method inside a Java class. Therefore, when processing large JSON data volumes, the code length must be less than 65536 characters, since the Script mediator converts the payload into a Java object. However, you can use the following alternative options to process large JSON data volumes.
+    The Micro Integrator uses Rhino engine to execute JavaScript. Rhino engine converts the script to a method inside a Java class. Therefore, when processing large JSON data volumes, the code length must be less than 65536 characters, since the Script mediator converts the payload into a Java object. However, you can use the following alternative options to process large JSON data volumes.
 
     -   Achieve the same functionality via a [Class mediator]({{base_path}}/reference/mediators/class-mediator) .
```
Comment (Mariangela) [inline en/docs/reference/mediators/script-mediator.md:8]:
```suggestion
    -   Achieve the same functionality via a [Class mediator]({{base_path}}/reference/mediators/class-mediator).
```

**Review by Mariangela — APPROVED** [review]

File: en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md]: Fixed.

File: en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md (hunk 2 above)
Comment (samgnaniah) [reply en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md]: Fixed.

**Pull request merged**